hydpy/models/gland/gland_model.py

```diff
@@ -0,0 +1,1614 @@
+
+
+class Calc_AE_V1(modeltools.Method):
+    """Calculate actual evaporation |AE| (for output only).
+
+    Basic equations:
+
+      :math:`AE = EI + ES`
+
+    Examples:
+
+        >>> from hydpy.models.gland import *
+        >>> from hydpy import pub
+        >>> parameterstep('1d')
+        >>> fluxes.ei = 8.
+        >>> fluxes.es = 1.978652
+        >>> model.calc_ae_v1()
+        >>> fluxes.ae
+        ae(9.978652)
+    """
+
+    REQUIREDSEQUENCES = (gland_fluxes.EI, gland_fluxes.ES)
+    RESULTSEQUENCES = (gland_fluxes.AE,)
+
+    @staticmethod
+    def __call__(model: modeltools.Model) -> None:
+        flu = model.sequences.fluxes.fastaccess
+        flu.ae = flu.ei + flu.es
+
+
+class Calc_Pr_V1(modeltools.Method):
+    """Calculate total quantity |PR| of water reaching the routing functions.
+
+    Basic equation:
+
+      :math:`PR = Perc + (PN - PS)`
+
+    Examples:
+
+        Example production store almost full, no rain:
+
+        >>> from hydpy.models.gland import *
+        >>> parameterstep('1d')
+        >>> fluxes.ps = 3.
+        >>> fluxes.perc = 10.
+        >>> fluxes.pn = 5.
+        >>> model.calc_pr_v1()
+        >>>
+        >>> fluxes.pr
+        pr(12.0)
+    """
+
+    REQUIREDSEQUENCES = (gland_fluxes.PS, gland_fluxes.PN, gland_fluxes.Perc)
+
+    RESULTSEQUENCES = (gland_fluxes.PR,)
+
+    @staticmethod
+    def __call__(model: modeltools.Model) -> None:
+        flu = model.sequences.fluxes.fastaccess
+        flu.pr = flu.perc + flu.pn - flu.ps
+
+
+class Calc_PR1_PR9_V1(modeltools.Method):
+    r"""Splitting |PR| into |PR1| and |PR9|.
+
+    Basic equations:
+
+      :math:`PR9 = 0.9 \cdot PR`
+
+      :math:`PR1 = PR - PR1`
+
+    Examples:
+
+        Example production store nearly full, no rain:
+
+        >>> from hydpy.models.gland import *
+        >>> parameterstep('1d')
+        >>> fluxes.pr = 10.0
+        >>> model.calc_pr1_pr9_v1()
+        >>> fluxes.pr9
+        pr9(9.0)
+        >>> fluxes.pr1
+        pr1(1.0)
+    """
+
+    REQUIREDSEQUENCES = (gland_fluxes.PR,)
+    RESULTSEQUENCES = (gland_fluxes.PR9, gland_fluxes.PR1)
+
+    @staticmethod
+    def __call__(model: modeltools.Model) -> None:
+        flu = model.sequences.fluxes.fastaccess
+        flu.pr9 = 0.9 * flu.pr
+        flu.pr1 = flu.pr - flu.pr9
+
+
+class Calc_Q9_RConcModel_V1(modeltools.Method):
+    """Let a submodel that follows the |RConcModel_V1| submodel interface calculate
+    runoff concentration."""
+
+    REQUIREDSEQUENCES = (gland_fluxes.PR9,)
+    RESULTSEQUENCES = (gland_fluxes.Q9,)
+
+    @staticmethod
+    def __call__(
+        model: modeltools.Model, submodel: rconcinterfaces.RConcModel_V1
+    ) -> None:
+        flu = model.sequences.fluxes.fastaccess
+        submodel.set_inflow(flu.pr9)
+        submodel.determine_outflow()
+        flu.q9 = submodel.get_outflow()
+
+
+class Calc_Q9_V1(modeltools.Method):
+    """Calculate the runofff concentration with |PR9| as input.
+
+    Examples:
```

Comment:
I replaced the doctest (also for Calc_Q1_V1 and Calc_Q10_V1). I don't like in your example that it is not demonstrated that directflow is usually calculated with "gr_uh2" und the routingstore is calculated with "gr_uh1".
I leave the topic open because of the issue.

Reply:
#144 is fixed by #145, so we can remove the mentioned line.

Reply:
One could mention the related options in the text and link to the corresponding doctests.  Still, I think this is not overly important because the application models' documentation contains this information. As you prefer.